i18n/vscode-language-pack-zh-hans/translations/main.i18n.json

```diff
@@ -3795,7 +3795,7 @@
 			"label": "问题",
 			"tooltip.1": "此文件存在 1 个问题",
 			"tooltip.N": "此文件存在 {0} 个问题",
-			"markers.showOnFile": "显示文件和文件夹上的错误和警告。"
+			"markers.showOnFile": "在文件和文件夹上显示错误和警告。"
```

Comment:
This settings control whether it shows errors/warnings on File/Folder items in File Explorer. Its shows them regardlessly in Problem panel. Same applies to the Outline one below.